opendebates/templates/base.html

```diff
@@ -65,14 +65,14 @@
     </div>
     <div class="login-container{% if request.user.is_authenticated %} login-container-authenticated{% endif %}">
       {% if request.user.is_authenticated %}
-        <a href="{# url 'user_profile' #}">
+        <a href="{# url 'user_profile' SITE_MODE.prefix #}">
```

Comment:
Might be worth checking what the intended behavior is here... a link with no `href` seems odd (why not just remove the link altogether?)...